database.yml

```diff
@@ -0,0 +1,18 @@
+default: &default
```

Comment:
This file seems misplaced. I'd expect it to be in something like `config` folder.